pkg/asset/machines/worker.go

```diff
@@ -156,9 +156,9 @@ func defaultOvirtMachinePoolPlatform() ovirttypes.MachinePool {
 
 func defaultVSphereMachinePoolPlatform() vspheretypes.MachinePool {
 	return vspheretypes.MachinePool{
-		NumCPUs:           2,
-		NumCoresPerSocket: 2,
-		MemoryMiB:         8192,
+		NumCPUs:           4,
+		NumCoresPerSocket: 1,
```

Comment:
Above it is 4 `NumCoresPerSocket` and here it is 1. Which is right?

Reply:
4 cpus and 1 cores per socket is right. I'll fix it

Reply:
Changed it to set masters to 4 vcpus and 4 numcores and workers to 4 vcpus and 1 numcores